cursorless-talon-dev/src/cursorless_test.talon

```diff
@@ -19,3 +19,5 @@ test api wrap with snippet <user.cursorless_target>:
     user.cursorless_wrap_with_snippet("Hello, $foo!  My name is $bar!", cursorless_target, "foo", "statement")
 test api wrap with snippet by name <user.cursorless_target>:
     user.cursorless_wrap_with_snippet_by_name("functionDeclaration", "body", cursorless_target)
+test snippet make <user.cursorless_target>:
+    user.private_cursorless_make_snippet_test(cursorless_target)
```

Comment:
this is for recording a snippet make test; the commands in this file are only to be activated during Talon spoken form tests

Reply:
Okay I will move it back